packages/react-components/src/components/PromoBannerV2/PromoBannerV2.stories.css

```diff
@@ -38,3 +38,19 @@
 .text {
   font-size: 24px;
 }
+
+.dark-promo-banner {
+  max-width: 800px;
+}
+
+.dark-promo-header {
+  margin: 0 0 var(--spacing-4) 0;
+}
+
+.dark-promo-additional-content {
+  align-items: center;
+
+  img {
```

Comment:
Are we sure that we want to force the radiuses on the img? 🤔 
What about cases where this will be unexpected?

Reply:
Do we do not want to do that, so that's why radius is added in custom story styles and not in the component ;)